infrastructure/scripts/test_gpt5_api_access.py

```
@@ -0,0 +1,309 @@
+            if "model" in error_msg.lower() and (
+                "not found" in error_msg.lower() or "does not exist" in error_msg.lower()
+            ):
+                result["error_type"] = "model_not_found"
+            else:
+                result["error_type"] = "api_error"
+
+            logger.warning("model_test_failed", model=model_name, error=error_msg)
+
+        return result
+
+    async def test_fallback_mechanism(self, client: GPT5MigrationClient) -> bool:
```

Ruff (0.14.4)

Unused method argument: `client`

(ARG002)